### ISSUE TITLE
ECP Protocol & Discord Voting

### ISSUE THREAD

**Issue by hashratez** [Ether1Project/ECPs#5]
As we transition from the Ether1 Management Team to the New Provisional Council (PC) there needs to be a transitory framework by which things can get done and the Community can provide bottom-up direction.

Our Discord Server is the primary property for project collaboration thus it should be used as a simple method for Community Contribution.

There should be a standard protocol for ECPs and voting.

When a new ECP is created on GithHub a notice should also be posted in the Governance Channel in Discord for Community awareness.  

The clock starts with the time date/stamp of GH creation:
1. Each ECP has a 14 day life-span for comments before voting.  While live banter can take place on Discord, serious amendments pros/cons to the ECP should be posted under it on GitHub.
2. At the end of the 14 days, the ECP comment period has ended.  The ECP will be updated by the author within 72 hours for Final Draft.
3. The final draft will be locked on Github and will go to vote on Discord under the Voting Channel, there is a 7 day voting window on the ECP.
4. If the ECP is passed and requires funding the ECP will be posted on the ECP Funding Dashboard.

So in summary
Days 1-14: Comments
Days 15-17: Amendments
Days 18-25: Voting

There is no cost to this ECP.

**Comment by Bigpiggy01** [comment]
I generally like the concept, however 14 days with people potentially taking holidays etc is on the short side. 30 days would seem a significantly fairer voting/engagement window.

**Comment by hashratez** [comment]
The project will only move as fast as the timeline set.  The PC can set 1 day, 1 hour or 6 weeks.  If we want move fast shorter framework, we want to move slow set it longer--but it needs to be set.  If there is not a process or a timeline associated with it, nothing will happen...

**Comment by Bigpiggy01** [comment]
Agreed but at the same time we need make sure that we keep the playing field level. an alternative could be adding a mailing list so no one can come and say they "didn't know" and cry on that account. Meaning a shorter time frame backed by extra communication.

**Comment by Primate411** [comment]
Voting on ECP's shouldn't be possible without certain conditions being met.  We're talking simple things like making sure they're in the correct format, contain a unique identifying number and all the required fields for tracking, indexing, etc.

Putting an arbitrary time on ECP's limits possibilities for the governance council in artificial way where it's not necessary. Council candidate @Bigpiggy01 pointed out one possible issue with it and there are many others, just imagine for yourself.  Have you ever taken a 2 week vacation without internet access? Think about it the other way around too.  If everyone is available and of the same mind about something, it shouldn't take 20+ days to get it done.

**Comment by Primate411** [comment]
Let's use this thread to figure out the best, least complicated and most flexible way to this process.

First and foremost what kind of proposal format do you think is best? Have you had good experiences with a crypto proposal process already in the past? At the very least we need to give each ECP a 4 digit tracking number.

**Comment by hashratez** [comment]
Having no timeline only leads to inaction.  I don't care if it is 2 hours, 2 days, 2 weeks or even 2 months.  But there needs to be finality for progress.

**Comment by hashratez** [comment]
Here is a great excerpt from the PIXV DAO implementation. I think is a good place to start--best would be to automate this in a Smart Contract for hands off governance.  Now I know there is NO WAY this can happen week one, or maybe not even year one.  BUT if we could indeed achieve this it would be huge.  [https://bops.cc/share/wl/?id=8ERSGbY3d573iRy1uQUxRevGedCRPoHK](url)

**Comment by Dev-JamesR** [comment]
I think you are getting way over the bow of the boat on all these ECPs. There are some great ideas and some I don't agree with but to be an effective council you need to go back to the basics before even thinking about some of these in depth ideas. First, how are you going to get people to build this stuff? I haven't seen any proposals or outlines on the biggest problem of all, people. How will you guarantee that anyone will actually work on any of these proposals? With an all volunteer force there are no guarantees anyone follows you. You put a deadline in place and most volunteers will just ignore you or at the very least end up demoralized and fade away. You either need to build a passionate following or you could chose to pay people, so then if they don't produce or follow the councils plan they don't get paid but then where would these funds come from then? We have had zero success community fundraising in the past, how can you fix this? ECPs #1-100 should be related to how you can get people, contributions and funding. We can't even get most people to send a few tweets per day outside of the core community/group. How do you expect to have massive projects like this worked on? If you want stuff done, that needs to be solved. Proposing and voting on all these other proposals along the way will ultimately give you the inaction you are against, then community morale tanks because you have 50 outstanding ECPs with nobody to actually do anything with them, that is unless you address the underlying resource issues first. Just my two cents, from two years of running the project. Too many cooks and opinions in the kitchen with too few people actually putting anything in the oven, or the microwave at least.

**Comment by hashratez** [comment]
The basic follow up concept was an ECP could not be proposed unless it was funded and had means to complete.  This was to avoid "dream ECPs".  You have an idea, great.  It needs to be scoped and paid for.
